packages/api/get-service-stops.ts

```diff
@@ -19,8 +19,29 @@ export const getQueryInput = (event: APIGatewayEvent): ServiceStopsQueryInput =>
         throw new ClientError("Provided service ID is not valid");
     }
 
+    const stopTypes = pathParameters.stopTypes || "";
+    const stopTypesArray = stopTypes
+        .split(",")
+        .filter((stop) => stop)
+        .map((stop) => stop.trim());
+
+    const modes = pathParameters.modes || "";
+    const modesArray = modes
+        .split(",")
+        .filter((mode) => mode)
+        .map((mode) => mode.trim());
+
+    const filteredModesArray = modesArray.filter(isValidMode);
+
+    if (filteredModesArray.length !== modesArray.length) {
+        throw new ClientError("Invalid mode provided");
+    }
+
     return {
         serviceId: Number(serviceId),
+        ...(pathParameters?.busStopType ? { busStopType: pathParameters.busStopType } : {}),
+        ...(filteredModesArray && filteredModesArray.length > 0 ? { modes: filteredModesArray } : {}),
+        ...(stopTypesArray && stopTypesArray.length > 0 ? { stopTypes: stopTypesArray } : {}),
```

Comment:
`busStopType` is not read from pathParameters and passed as input to client

Reply:
if iz lins 42